src/main.cc

```diff
@@ -1164,8 +1164,16 @@ int main(int argc, char* argv[])
                 }
             }
             String new_files;
-            for (auto name : files)
+            for (auto name : files) {
                 new_files += format("edit '{}';", escape(real_path(name), "'", '\\'));
```

Comment:
It'd be cleaner if you passed the coordinate to the `edit` command directly.

Reply:
Yeah, thanks!